app/models/edo_oracle/financial_aid/queries.rb

```diff
@@ -5,21 +5,39 @@ class Queries < Connection
       include ClassLogger
       include Concerns::QueryHelper
 
-      def self.get_housing(person_id, aid_year)
+      def self.get_housing(person_id, aid_year:, effective_date: Time.zone.today.in_time_zone.to_date)
```

Comment:
@marcpechaitis - Where is the `aid_year` value here?

Reply:
Oh I see. Ruby 2.1 introduced required keyword arguments, which are [defined with a trailing colon](https://thoughtbot.com/blog/ruby-2-keyword-arguments).